code/SudoModeController.php

```diff
@@ -91,7 +89,7 @@ public function activate(HTTPRequest $request): HTTPResponse
             return $this->jsonResponse([
                 'result' => false,
                 'message' => _t(__CLASS__ . '.INVALID', 'Incorrect password'),
-            ]);
+            ], 400);
```

Comment:
Unrelated refactoring - this is an error, so should have an error response code!  There's no context in the original PR that created this class for why it was a 200. It looks like it was an oversight.

Reply:
I hate HTTP response codes lol.
Updated to 401